typings/solid-auth-client.d.ts

```diff
@@ -0,0 +1,30 @@
+export interface SolidSession {
```

Comment:
Some amazing person actually submitted type definitions to DT for solid-auth-client! :)

https://www.npmjs.com/package/@types/solid-auth-client